david_vicklund/gulpfile.js

```diff
@@ -23,6 +28,29 @@ gulp.task('jshint:test', function() {
   .pipe(jshint.reporter('jshint-stylish'));
 });
 
+gulp.task('webpack:test', function() {
+  return gulp.src('test/client/test_entry.js')
+    .pipe(webpack({
+      output: { 
+        filename: 'test_bundle.js'
+      }
+    }))
+    .pipe(gulp.dest('test/client/'));
+});
+
+// CSS tasks
+gulp.task('minify:scss', function() {
+  gulp.src('app/scss/*.scss')
+    .pipe(sass())
+    .pipe(gulp.dest('app/css/'));
+  return gulp.src('app/css/**/*.css')
+    .pipe(sourcemaps.init())
+    .pipe(concatCss('styles.min.css'))
+    .pipe(minifyCss())
+    .pipe(sourcemaps.write())
+    .pipe(gulp.dest('build/'));
+});
```

Comment:
Yeah, that was sort of a side effect of doing the scss assignment before
the vanilla CSS

On Thu, Dec 10, 2015, 2:01 PM Paul Laskowski <notifications@github.com>
wrote:

> In david_vicklund/gulpfile.js
> <https://github.com/dvicklund/mongo_backed_rest_api/pull/3#discussion_r47292847>
> :
>
> > +    }))
> > +    .pipe(gulp.dest('test/client/'
>
> ));
> > +});
> > +
> > +// CSS tasks
> > +gulp.task('minify:scss', function() {
> > +  gulp.src('app/scss/*.scss')
> > +    .pipe(sass())
> > +    .pipe(gulp.dest('app/css/'));
> > +  return gulp.src('app/css/**/*.css')
> > +    .pipe(sourcemaps.init())
> > +    .pipe(concatCss('styles.min.css'))
> > +    .pipe(minifyCss())
> > +    .pipe(sourcemaps.write())
> > +    .pipe(gulp.dest('build/'));
> > +});
>
> It looks like you've combined the CSS gulp task with the SCSS gulp task.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dvicklund/mongo_backed_rest_api/pull/3/files#r47292847>
> .
>